tests/multiproject_test.js

```diff
@@ -0,0 +1,258 @@
+const checkoutCID = '6b02a62df319:47a781df83';
+
+Scenario('Verify changes when both experiments matching', async ({ I }) => {
+    let cssAsset = null;
+    //Act
+    I.amOnPage('/index.html');
+    
+    const scripts = await I.grabAttributeFromAll(locate('//head').find('script'),'src');
+    await scripts.forEach(element => {
+        if(element)
+        if(element.includes('evolv')){
+            allocations = getAllocations();
+        }
+    });
+    
+    const links = await I.grabAttributeFromAll(locate('//head').find('link'),'href');
+   
+    await links.forEach(element => {
+        if(element)
+        if(element.includes('evolv')){
+            cssAsset = element;
+        }
+    });
+    if(!cssAsset){
+    
+        throw new Error('links does not contain evolv assets');  
+    }
+
+    const locator = await getCssLocator(cssAsset);
+    const css = await getCssAttribute(cssAsset);
+   
+    let confirmations = await I.executeScript(async () => {
+        let confirmationsArray = [];
+        evolv.context.remoteContext.confirmations.forEach(element => {
+            confirmationsArray.push(element.cid);
+        })
+        return confirmationsArray;
+    });
+    //Assert
+    await locator.forEach(async element => {
+        I.waitForElement(element);
+        let cssAttr = await I.grabCssPropertyFrom(element, css[locator.indexOf(element)].attr);
+        I.assertContain(cssAttr, convert.keyword.rgb(css[locator.indexOf(element)].value).join(', ').toString());
+    });
+    
+    await I.assertContain(confirmations, learnMoreCID);
+    await I.assertContain(confirmations, checkoutCID);
+    
+});
+
+Scenario('Verify changes when first experiment matching', async ({ I }) => {
+    let cssAsset = null;
+    //Act
+    I.amOnPage('/clone.html');
+    const scripts = await I.grabAttributeFromAll(locate('//head').find('script'),'src');
+    await scripts.forEach(element => {
+        if(element)
+        if(element.includes('evolv')){
+            allocations = getAllocations();
+        }
+    });
+    
+    const links = await I.grabAttributeFromAll(locate('//head').find('link'),'href');
+   
+    await links.forEach(element => {
+        if(element)
+        if(element.includes('evolv')){
+            cssAsset = element;
+        }
+    });
+    if(!cssAsset){
+    
+        throw new Error('links does not contain evolv assets');  
+    }
+
+    const locator = await getCssLocator(cssAsset);
+    const css = await getCssAttribute(cssAsset);
+    let confirmations = await I.executeScript(async () => {
+        let confirmationsArray = [];
+        evolv.context.remoteContext.confirmations.forEach(element => {
+            confirmationsArray.push(element.cid);
+        })
+        return confirmationsArray;
+    });
+    //Assert
+    
+    I.waitForElement(locator[0]);
+    let cssAttr = await I.grabCssPropertyFrom(locator[0], css[0].attr);
+    I.assertContain(cssAttr, convert.keyword.rgb(css[0].value).join(', ').toString());
```

Comment:
We should check the non matching experiment changes are not applied

Reply:
resolved